Fix minor documentation typos (#2256)
kubeflow/website: content/en/docs/pipelines/tutorials/benchmark-examples.md

Use the following instructions to run the benchmark script on your Kubeflow
Pipelines cluster.

> 

Reviewer: Line 131:

- Add 1 indefinite and 1 definite article

```diff
- it is also possible to show KDE (Kernel Density Estimate) plot. If a KDE plot is
+ it is also possible to show a KDE (Kernel Density Estimate) plot. If the KDE plot is
```

> 1. Download the [run_service_api.ipynb](https://storage.googleapis.com/ml-pipeline/sample-benchmark/run_service_api.ipynb)

Reviewer: If you have time, maybe we can also use "code font" for file names (e.g. lines 67 and others - `.ipynb` notebooks and YAML files (search for `.yaml`):

```suggestion
1. Download the [`run_service_api.ipynb`](https://storage.googleapis.com/ml-pipeline/sample-benchmark/run_service_api.ipynb)
```

(Reference - [Google docs style guide: Code in text](https://developers.google.com/style/code-in-text#some-specific-items-to-put-in-code-font)).

   benchmark script in your Jupyter notebook environment.

> 1. Open [run_service_api.ipynb](https://storage.googleapis.com/ml-pipeline/sample-benchmark/run_service_api.ipynb)

Reviewer: We can use the "code font" for file names (e.g. `.ipynb` notebooks and YAML files (search for `.yaml`)):

```suggestion
1. Open [`run_service_api.ipynb`](https://storage.googleapis.com/ml-pipeline/sample-benchmark/run_service_api.ipynb)
```

(Reference - [Google docs style guide: Code in text](https://developers.google.com/style/code-in-text#some-specific-items-to-put-in-code-font)).

Reviewer: - Line 87:

```diff
-      [taxi_updated_pool.yaml](https://storage.googleapis.com/ml-pipeline/sample-benchmark/taxi_updated_pool.yaml)
+      [`taxi_updated_pool.yaml`](https://storage.googleapis.com/ml-pipeline/sample-benchmark/taxi_updated_pool.yaml)
```

- Lines 111-112:

```diff
- [`run_service_api.ipynb`](https://storage.googleapis.com/ml-pipeline/sample-benchmark/run_service_api.ipynb)
+ [`run_service_api.ipynb`](https://storage.googleapis.com/ml-pipeline/sample-benchmark/run_service_api.ipynb)
- using the [taxi_updated_pool.yaml](https://storage.googleapis.com/ml-pipeline/sample-benchmark/taxi_updated_pool.yaml)
+ using the [`taxi_updated_pool.yaml`](https://storage.googleapis.com/ml-pipeline/sample-benchmark/taxi_updated_pool.yaml)

and the y-axis shows how many runs fall into certain status respectively.

In a distribution plot, both histogram plot and rug plot are shown. In addition,
it is also possible to show a KDE (Kernel Density Estimate) plot. If the KDE plot is

> desirable, use 'kde=True' in the distplot() method.

Reviewer: Can you check the use of "code font" - singe backticks (vs quotation marks) for parameters and functions/methods, etc?  (Reference - [Google docs style guide: Code in text](https://developers.google.com/style/code-in-text#some-specific-items-to-put-in-code-font)).

Line 131:

```suggestion
desirable, use `kde=True` in the `distplot()` method.
```

Reviewer: - Line 50:

```diff
- CREATE, GET, DELETE, and LIST methods of the pipeline, pipeline version
+ `CREATE`, `GET`, `DELETE`, and `LIST` methods of the pipeline, pipeline version
```

- Line 76:

```diff
-   1. Records the latency of CREATE, GET, DELETE.
+   1. Records the latency of `CREATE`, `GET`, `DELETE`.
```

- Line 80:

```diff
1. In the benchmark script, enter the correct values for host,
-   pipeline_file_url, num_runs, run_status_polling_interval_sec in the
+   `pipeline_file_url`, `num_runs`, `run_status_polling_interval_sec` in the
   benchmark script.
```

- Lines 92, 94:

```diff
-       **NOTE**: Do not use the value 'https://storage.cloud.google.com/ml-pipeline/sample-benchmark/taxi_updated_pool.yaml'
+      **NOTE**: Do not use the value `https://storage.cloud.google.com/ml-pipeline/sample-benchmark/taxi_updated_pool.yaml`
     when running your benchmarks. Addresses that start with
-      'storage.cloud.google.com' incur a redirect that doesn't work well with
+      `storage.cloud.google.com` incur a redirect that doesn't work well with
        ...
```

- Line 114:

```diff
- Each node pool has three nodes of machine type 'n1-standard-8'.
+ Each node pool has three nodes of machine type `n1-standard-8`.
```
